ui/components/app/transaction-list-item/transaction-list-item.component.js

```diff
@@ -165,9 +190,31 @@ function TransactionListItemInner({
     hasCancelled,
     retryTransaction,
     cancelTransaction,
+    ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
+    isCustodian,
+    ///: END:ONLY_INCLUDE_IN
   ]);
 
-  const showCancelButton = !hasCancelled && isPending && !isUnapproved;
+  let showCancelButton = !hasCancelled && isPending && !isUnapproved;
+
+  ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
+  showCancelButton = showCancelButton && !isCustodian;
+  const PENDING_COLOR = '#6A737D';
+  const OK_COLOR = '#2F80ED';
+  const FAIL_COLOR = '#D73A49';
```

Comment:
Why use these instead of the var(--color-error-default) etc? similar to how TransactionIcon does it?

Reply:
We may change the color of icons based on theme and hardcoding these hex values makes the component brittle to theme changes in the future.

Reply:
Correct, will update this 👍🏼